components/menu/menu-navigation-style.tsx

```diff
@@ -48,7 +48,7 @@ function MenuItem({ item }: { item: MenuItem }) {
 function SingleItem({ item }: { item: MenuItem }) {
   return (
     <NavigationMenuItem>
-      <Link href={item.href} legacyBehavior passHref>
+      <HostPrefixedLink href={item.href}>
         <NavigationMenuLink
```

Comment:
`<NavigationMenuLink>` is also an anchor (`<a>`) tag. Nested anchors will cause a hydration error